replay/models/cql.py

```diff
@@ -0,0 +1,217 @@
+"""
+Using CQL implementation from d3rlpy
+For 'alpha' version PySpark DataFrame converts to Pandas
+"""
+
+from typing import Any, Dict, Optional
+
+import d3rlpy.algos.cql as CQL_d3rlpy
+import numpy as np
+import pandas as pd
+import pyspark.sql
+import torch
+import torch.nn as nn
+from d3rlpy.dataset import MDPDataset
+from pandas import DataFrame
+
+from replay.data_preparator import DataPreparator
+from replay.models.base_torch_rec import TorchRecommender
+
+
+class CQL(TorchRecommender):
```

Comment:
consider inheriting from Recommender since you are not using the Torch Recommender functionality